### DISCUSSION TITLE
Does NNX support JAX checkify

### DISCUSSION THREAD

**Discussion by cemlyn007** [google/flax#4378] [Q&A] [answered]
With the new NNX stateful API, does anyone know how to use checkify with it?

Kind regards,
Cemlyn

**Comment by cgarciae** [comment, marked as answer]
Hey, you can always use the [Functional API](https://flax.readthedocs.io/en/latest/nnx_basics.html#the-flax-functional-api) to interact with JAX directly in case we don't provide the functionality yet. However this is very useful and easy to implement so I've added it in #4381.

**Reply by cemlyn007** [reply]
Thank you so much @cgarciae! Hope you a great weekend!
